### PR TITLE
catalog 1

### PR DESCRIPTION
**Pull request by BiletskyiVolodymyr**
- [DEMO LINK](https://<BiletskyiVolodymyr>.github.io/layout_catalog/)
- [TEST REPORT LINK](https://<BiletskyiVolodymyr>.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Frontend practice with catalog page
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://<BiletskyiVolodymyr>.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://<BiletskyiVolodymyr>.github.io/layout_catalog/report/html_report/)
```
Comment (Anastasiia-Svintsova) [inline readme.md:4]: Fix your links. And there is smth wrong with your deployment
image

```suggestion
- [DEMO LINK](https://BiletskyiVolodymyr.github.io/layout_catalog/)
- [TEST REPORT LINK](https://BiletskyiVolodymyr.github.io/layout_catalog/report/html_report/)
```

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,435 @@
...
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link"
+              data-qa="nav-hover"
+              href="#Laptops_&_Computers">Laptops & Computers</a>
```
Comment (Anastasiia-Svintsova) [inline src/index.html:35]:
```suggestion
            <a 
              class="nav__link"
              data-qa="nav-hover"
              href="#Laptops_&_Computers"
            >
              Laptops & Computers
            </a>
```

File: src/index.html [hunk 3]
```diff
@@ -5,8 +5,435 @@
...
+            </div>
+          </div>
+    
+          <a 
+            class="product-card__button"
+            href="#"
+            data-qa="card-hover"
+          >
+          buy
+          </a>
```
Comment (Anastasiia-Svintsova) [inline src/index.html:104]:
```suggestion
          >
            buy
          </a>
```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]

**Comment by BiletskyiVolodymyr** [comment]
- [DEMO LINK](https://BiletskyiVolodymyr.github.io/layout_catalog/)
- [TEST REPORT LINK](https://BiletskyiVolodymyr.github.io/layout_catalog/report/html_report/)

File: src/styles/variables.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+$grey-color: #616070;
+$black-color: #060b35;
+$blue-color: #00acdc;
+$white-color: #fff;
+
+$catalog-p-1: 50px;
```
Comment (agniya-i) [inline src/styles/variables.scss:6]:
```suggestion
$catalog-padding-top: 50px;
```
it's better to give more details in variables names

File: src/index.html [hunk 5]
```diff
@@ -5,8 +5,439 @@
...
+
+
+        <div class="product-card">
+          <img
+            class="product-card__img" 
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+    
+          <div class="product-card__name">
```
Comment (agniya-i) [inline src/index.html:260]:
```suggestion
          <h2 class="product-card__name">
```
use semantic tags

File: src/index.html [hunk 6]
```diff
@@ -5,8 +5,439 @@
...
+            APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+          </div>
+    
+          <div class="product-card__code">
```
Comment (agniya-i) [inline src/index.html:264]:
```suggestion
          <p class="product-card__code">
```

File: src/index.html [hunk 7]
```diff
@@ -5,8 +5,439 @@
...
+              <div class="stars__star"></div>
+            </div>
+    
+            <div class="product-card__responce">
+              Reviews: 5
+            </div>
+          </div>
+    
+          <div class="product-card__price">
+            <div class="product-card__title">
```
Comment (agniya-i) [inline src/index.html:283]:
```suggestion
            <p class="product-card__title">
```
same here

File: src/index.html [hunk 8]
```diff
@@ -5,8 +5,439 @@
...
```
Comment (agniya-i) [inline src/index.html:330]:
```suggestion
            <div class="product-card__price-title">
```
`product-card__title` - seems that is title of the whole card, not only the price

**Review by agniya-i — CHANGES_REQUESTED** [review]

File: src/styles/blocks/header.scss [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+    text-decoration: none;
+    color: $black-color;
+    padding: 22.5px 0;
+    display: flex;
+    align-self: center;
+    margin-right: 20px;
+  }
+  
+  .nav__link:hover {
+    color: #00acdc;
```
Comment (Anastasiia-Svintsova) [inline src/styles/blocks/header.scss:39]: You have this color in variables, so use it

File: src/styles/blocks/header.scss [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+  }
+  
+  .is-active {
+    color: #00acdc;
+    position: relative;
+  }
+  
+  .nav__item .is-active {
+    color: #00acdc;
+  }
```
Comment (Anastasiia-Svintsova) [inline src/styles/blocks/header.scss:53]: You already have the same style for `.nav__link:hover`. Combine them, please

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- The card shouldn't change height on the button hover

https://user-images.githubusercontent.com/99403198/213869716-e445b45e-ce3e-4805-9bcc-2da5997dbf60.mov

**Pull request closed without merging**